lib/charms/kubernetes_charm_libraries/v0/multus.py

```diff
@@ -619,3 +662,7 @@ def _on_remove(self, event: RemoveEvent) -> None:
                 self.kubernetes.delete_network_attachment_definition(
                     name=network_attachment_definition.metadata.name  # type: ignore[union-attr]
                 )
+
+    def delete_pod(self) -> None:
+        """Delete the pod."""
+        self.kubernetes.delete_pod(self._pod)
```

Comment:
Should this change be part of this PR?

Reply:
This belongs to another PR, it should be removed.